### DISCUSSION TITLE
Spelunkery

### DISCUSSION THREAD

**Discussion by WenXin20** [DynamicTreesTeam/DynamicTrees#932] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/spelunkery/files/5619021
- Minecraft 1.20.1
- 2 mushrooms in the Deep Dark

![image](https://github.com/user-attachments/assets/796c426f-bdab-4444-9687-4d9df9d967c9)

**Comment by gokufunnygirl** [comment]
The mod added more mushrooms since this post
